src/LoggerSimple.h

```diff
@@ -2,33 +2,59 @@
 // Logging free functions defined in Logger.cpp
 #pragma once
 
-// python style format specified logging
-
-extern void FLOG(const char * a_Format, fmt::ArgList a_ArgList);
-FMT_VARIADIC(void, FLOG, const char *)
+#include "Logger.h"
```

Comment:
Literally the entire point of `LoggerSimple.h` is that it __doesn't__ include `Logger.h`.

Reply:
Depending on how the above discussion goes (meta!) using templates may require it.